app/src/main/java/org/p2p/wallet/send/model/SendFeeTotal.kt

```diff
@@ -35,6 +37,33 @@ class SendFeeTotal constructor(
     val interestBearingPercent: BigDecimal? = null
 ) : Parcelable {
 
+    companion object {
+        fun createEmpty(): SendFeeTotal = SendFeeTotal(
```

Comment:
mb just val EMPTY?

Reply:
No reason to do this; this code is unused in fact, and allocating static storage for it is pointless